[2025-06-30T10:31:36.447+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-30T10:31:36.453+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_dag.fetch_covid_datas scheduled__2025-06-29T00:00:00+00:00 [queued]>
[2025-06-30T10:31:36.456+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_dag.fetch_covid_datas scheduled__2025-06-29T00:00:00+00:00 [queued]>
[2025-06-30T10:31:36.456+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 1
[2025-06-30T10:31:36.562+0000] {taskinstance.py:2907} INFO - Executing <Task(PythonOperator): fetch_covid_datas> on 2025-06-29 00:00:00+00:00
[2025-06-30T10:31:36.565+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=124) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-30T10:31:36.567+0000] {standard_task_runner.py:72} INFO - Started process 126 to run task
[2025-06-30T10:31:36.566+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'covid_dag', 'fetch_covid_datas', 'scheduled__2025-06-29T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/covid-dag.py', '--cfg-path', '/tmp/tmpazyqfuk7']
[2025-06-30T10:31:36.568+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask fetch_covid_datas
[2025-06-30T10:31:36.587+0000] {task_command.py:467} INFO - Running <TaskInstance: covid_dag.fetch_covid_datas scheduled__2025-06-29T00:00:00+00:00 [running]> on host 8b34973e9eba
[2025-06-30T10:31:36.615+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_dag' AIRFLOW_CTX_TASK_ID='fetch_covid_datas' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-29T00:00:00+00:00'
[2025-06-30T10:31:36.615+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-06-30T10:31:36.633+0000] {covid-dag.py:22} INFO - Fetching COVID data
[2025-06-30T10:31:37.233+0000] {warnings.py:112} WARNING - /opt/***/dags/covid-dag.py:24: DtypeWarning: Columns (0) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(ENDPOINT)

[2025-06-30T10:31:37.764+0000] {covid-dag.py:31} INFO - Saved COVID data to ./data/raw-datas-2025-06-30.csv
[2025-06-30T10:31:37.766+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-30T10:31:37.770+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-06-30T10:31:37.770+0000] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=covid_dag, task_id=fetch_covid_datas, run_id=scheduled__2025-06-29T00:00:00+00:00, execution_date=20250629T000000, start_date=20250630T103136, end_date=20250630T103137
[2025-06-30T10:31:37.788+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-30T10:31:37.802+0000] {taskinstance.py:3924} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-30T10:31:37.804+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
